### PR TITLE
added one wire plug-in

### PR DESCRIPTION
**Pull request by Boernsman**

### CODE REVIEW

File: plugins/deviceplugins/raspberryio [hunk 1]
```diff
@@ -0,0 +1 @@
+Subproject commit 1d598aed3a0e4ab83ea6a98622798345c18a807d
```
Comment (t-mon) [inline plugins/deviceplugins/raspberryio]: What's this? :)

**Comment by t-mon** [comment]
Jenkins, ok to test.

**Comment by t-mon** [comment]
Jenkins, retest this please.

**Comment by mzanetti** [comment]
Plugins have moved to a new repository. If this is still to be merged, please file a pull request to the new plugin repository.

**Pull request closed without merging**